prow/cluster/hook_deployment.yaml

```diff
@@ -30,10 +30,10 @@ spec:
       labels:
         app: hook
     spec:
-      terminationGracePeriodSeconds: 30
+      terminationGracePeriodSeconds: 180
```

Comment:
It doesn't seem that SIGTERM is handled by hook. Actually it's not handled at all today either by prow or the munger. Do we want a follow-up issue to support graceful shutdown?